Dockerfile.py3

```diff
@@ -127,9 +80,12 @@ RUN pip2 install --upgrade Cython==0.28.6
 
 WORKDIR ${WORK_DIR}
 COPY --chown=user:user . ${WORK_DIR}
-RUN chown --recursive ${USER} ${ANDROID_SDK_HOME}
+RUN mkdir ${ANDROID_HOME} && chown --recursive ${USER} ${ANDROID_HOME}
 USER ${USER}
 
+# Download and install android's NDK/SDK
+RUN make -f ci/makefiles/android.mk target_os=linux
```

Comment:
Nice! I'm just wondering, this layer would be cached entirely then correct? This is fine, I wasn't sure if it would be cached or not

Reply:
> ...this layer would be cached entirely then correct? This is fine, I wasn't sure...

I hope so...but I don't know for sure...so...:no_mouth: